test/unit/email_rate_limit_test.rb

```diff
@@ -13,9 +9,9 @@ def seed_array
     invalid_values        = last_clicked_days_ago.to_a - valid_values
     day_ago               = rand(last_clicked_days_ago)
     clicked_ago           = valid_values.sample
-    assert EmailRateLimit.new(day_ago).now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
+    assert_true { EmailRateLimit.new(day_ago).now?(clicked_ago) }
```

Comment:
I was looking for a way to dry up the assertions (and specifically the description of each, which effectively duplicate the method call). I'm a MiniTest noob (I've almost exclusively used rspec); one day I'll get around to reading the docs, but in the meantime I was curious of how I could essentially run some code and then, if it fails, print out what it was that failed. Whether using `assert foo?` or `assert_equal true, foo?`, the error messages appear to be pretty useless. By passing the block to the `assert_true` method I wrote (see below for implementation), I can use an error message that includes the contents of the block (so the error message is useful, reading `"Expected EmailRateLimit.new(5).now?(10) to be true"`). If there's a better way to do this, I'm all ears.